easycv/pipeline.py

```diff
@@ -35,19 +38,34 @@ def __init__(self, source, name=None):
             self._transforms = deepcopy(source)
 
         elif isinstance(source, str) and os.path.isfile(source):
-            try:
-                with open(source, "rb") as f:
-                    saved = pickle.load(f)
-                    if isinstance(saved, Pipeline):
-                        self._name = name if name else saved.name
-                        self._transforms = saved.transforms()
-                    else:
-                        raise InvalidPipelineInputSource()
-            except pickle.UnpicklingError:
-                raise InvalidPipelineInputSource() from None
+            with open(source, "r") as file:
+                source = yaml.load(file, Loader=yaml.FullLoader)
```

Comment:
Codacy found an issue: [Use of unsafe yaml load. Allows instantiation of arbitrary objects. Consider yaml.safe_load().](https://app.codacy.com/gh/easycv/easycv/pullRequest?prid=6255509)